gluon-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink

```diff
@@ -61,13 +61,22 @@ if [ "$(uci get wireguard.wireguard.enabled)" == "true" ] || [ "$(uci get wiregu
 		PEER=$(awk 'BEGIN{srand();print int(rand()*($NUMBER_OF_PEERS)+1) }')
 
 		endpoint="$(check_handshake "$(uci get wireguard.peer_$PEER.endpoint)")"
+		wg_listen_port="$( wg | grep "listening port" | awk '{print $3}' )"
```

Comment:
```suggestion
		wg_listen_port="$(wg show $MESH_VPN_IFACE | grep "listening port:" | awk '{print $3}')"
```

Reply:
Please no. Try this instead: `wg show $MESH_VPN_IFACE listen-port`